test/unit/getTableName-test.js

```diff
@@ -1,19 +0,0 @@
-require('require-self-ref')
-
```

Comment:
Huh, were there two of these files or is github doing weird things? Looks like one was deleted, one was renamed?